### DISCUSSION TITLE
Forking takes too much time and says took too much time to respond

### DISCUSSION THREAD

**Discussion by Shravan773** [community/community#47131] [Repositories] [answered]

**Comment by Shravan773** [comment]
solved!

**Reply by nonjee888** [reply]
can you please share what was the issue?

**Reply by viraj-s15** [reply]
Could you please share how you fixed it?

**Reply by weidehai** [reply]
i encounter same trouble, could you please share how you fixed it?

**Reply by Shravan773** [reply]
I closed and reloaded the website...it was solved after 5 tries...I don't know how😅

**Comment by Bejte** [comment]
same trouble and still continues for me as well. how did you solve it?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Shravan773 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by queenofcorgis** [comment, marked as answer]
This is now resolved - as we are not seeing these errors show up in our monitoring. Our monitors show this event happened February 12 between 6-8am ET. However, if you are still seeing this persist or have had this occur more recently, please let me know ❤️ .
